clash/force_of_will.tex

```diff
@@ -0,0 +1,101 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{Force of Will}{\images/berserk.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Shakajiub
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1237891632448278618}{Archon Studios Discord}
+
+\textit{Wild Dragons are terrorizing the countryside. Farmers are unable to grow their crops in peace, caravans are scared to move their goods. You must strike at their core and defeat the beasts. If by nothing else, then by sheer force of will.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This scenario is played over 12 rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 or 3
+
+\textbf{Starting Resources:}\par
+\resources{15}{2}{1}
+
+\textbf{Starting Income:}\par
+\resources{10}{2}{1}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item 2 × Few \svgunit{bronze} units with the \textit{highest} Recruitment cost
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling
+
+\textbf{Map tile Pool:} Each player takes 1 random Far (II--III) Map tile
+
+\textbf{Additional Bonus:} None
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map tiles and arrange them as shown in the scenario map layout:
+
+\textbf{For a 2-player scenario:}
+\begin{itemize}
+  \item 2 × Starting (I) Map tile
+  \item 2 × Near (IV--V) Map tile
+  \item 4 × Far (II--III) Map tile
+  \item 1 × Center (VI--VII) Map tile, which must contain the Dragon Utopia field
+\end{itemize}
+
+\textbf{For a 4-player scenario:}
+\begin{itemize}
+  \item 3 × Starting (I) Map tile
+  \item 3 × Near (IV--V) Map tile
+  \item 6 × Far (II--III) Map tile
+  \item 1 × Center (VI--VII) Map tile, which must contain the Dragon Utopia field
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+To win the scenario, a Hero must Flag the Dragon Utopia field.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+At the end of the \nth{12} Round, if there is no winner, all players lose the scenario.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{4} and \nth{8} Rounds:}
+\begin{itemize}
+  \item Remove all Black cubes from every Water Wheel and Windmill on the map.
+\end{itemize}
+
+\textbf{(2-P Scenario Only) \nth{5} and \nth{9} Rounds:}
+\begin{itemize}
+  \item The second player gains 1 \svg{movement}.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+During this scenario:
+
+\begin{itemize}
+  \item When a player Visits an Obelisk, they roll 2 \svg{treasure} and choose one to resolve.
+  \item A player may not use the Diplomacy card to skip Combat on the Dragon Utopia field.
+  \item When a Hero moves to the Dragon Utopia field, they must fight, depending on chosen difficulty, the following number of Neutral units:
+  \begin{itemize}[leftmargin=15pt]
```

Comment:
Why the `leftmargin`?

Reply:
To reduce the indentation. Makes sense if there are many bullet points, as it saves quite a lot of page real estate.

Reply:
Let's make it consistent then. This happens in other scenarios and it's used randomly. Titans' Stronghold even has different format on the same page

<img width="537" alt="Screenshot 2024-07-14 at 23 49 29" src="https://github.com/user-attachments/assets/eb30ddf1-43e3-4c8f-97a4-ae84b1209c90">

Reply:
There's also even deeper list.

<img width="256" alt="Screenshot 2024-07-14 at 23 53 04" src="https://github.com/user-attachments/assets/a223ea7f-50fe-4102-900c-9565adc81d7d">

It seems like we could have `15pt` margin for the sublist everywhere

Reply:
> I did that for Titan's Stronghold to make sure they fit in one line

Chances are other languages might not fit it anyway :)

Reply:
If it's a custom setting, it can be customized for every language.

Reply:
I was more worried about the design/layout. If English version was a tight fit, other languages could have issues (remember wyvern in the rulebook :D). Luckily, this page seems to have some spare space to allow for this

Reply:
I didn't do it because of the tight fit. It just looks better, as additional rules descriptions usually involve many bullet points. Besides, even if translations swell over to the next page, map layout can be positioned appropriately. I wouldn't consider it an issue now.

Reply:
https://github.com/qwrtln/Homm3BG-mission-book/issues/59